### ISSUE TITLE
Does mixup process still work during eval?

### ISSUE THREAD

**Issue by imhuay** [vikasverma1077/manifold_mixup#15]
I didn't find anything in the code like `nn.Dropout` to determine if it was in training.

**Comment by christopher-beckham** [comment]
I don't think it has anything to do with evaluation, since mixup is strictly a training procedure. At test time, there is no need to mix any points -- just feed in x and get the prediction for it as usual.

**Comment by imhuay** [comment]
Thank you for your answer.
